tests/unit/test_coordinate_utilities.py

```diff
@@ -543,3 +520,209 @@ def test_any_absent_dimension_variables(self):
                 self.test_varinfo, '/Soil_Moisture_Retrieval_Data_AM/variable_has_dim'
             )
             self.assertFalse(variable_has_fake_dims)
+
+    def test_get_row_col_geo_grid_points(self):
+        """Ensure that two valid lat/lon points returned by the method
+        with a set of lat/lon coordinates as input
+
+        """
```

Comment:
Nice tests. I think there are a couple of additional missing test-cases, though:

* There is only a single valid pixel.
* There are no valid pixels.

Reply:
Possibly also what happens when there are only valid points in a single row or single column (for geographic things).

Reply:
updated unit tests - https://github.com/nasa/harmony-opendap-subsetter/pull/18/commits/a3680c5f5a89f3443016c9bd0203b1e07d88e11b